CMake/FBCMakeParseArgs.cmake

```diff
@@ -0,0 +1,141 @@
+#
```

Comment:
this is an external cmake library. I copied it here with the other ones, but should we make it explicit and move it to something like `CMake/third-party/` ?